### PR TITLE
SEO-173296 WPF Chart Trendlines

### PR DESCRIPTION
**Pull request by Yvone-Atieno**
Hi @MallikaRK ,

Have renamed image folder, changed and image alt.

Regards,
Yvone.

### CODE REVIEW

**Review by MallikaRK — APPROVED** [review]
Approved

File: wpf/Charts/Trendlines.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ chart.Series.Add(scatterSeries);
 
 {% endtabs %}
 
-![Trendlines in WPF Chart](Trendlines_images/wpf-chart-trendlines.png)
+![Trendlines in WPF Chart](trendlines_images/wpf-chart-trendlines.png)
```
Comment (Girija-Rajendran) [inline wpf/Charts/Trendlines.md:71]: Changed this name in folder

Comment (Yvone-Atieno) [reply wpf/Charts/Trendlines.md:71]: the folder is changed

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine.

**Pull request merged**